Fix/vs code related issues (#250)
NordicSemiconductor/pc-nrfconnect-toolchain-manager: Changelog.md

## 1.4.2 - Unreleased

### Changed

-   Open VSCode Button now opens VSCode in the SDK Path

### Fixed

> -   Skipping installing vsCode extension will now open VScode and not reopen the

Reviewer: ```suggestion
-   Skipping installing vsCode extension will not reopen the
```

Author: ```suggestion
-   Skipping installing VS Code extension will now open VS Code and not reopen the same dialog.
```

> -   Skipping installing vsCode extension will now open VScode and not reopen the
>     same dialog

Author: ```suggestion
-   Skipping installing VS Code extension will not reopen the same dialog.
```
